src/DataAccess/Entities/Apprenticeship/Apprenticeship.cs

```diff
@@ -6,33 +6,22 @@ public class Apprenticeship
 {
 	public Apprenticeship()
 	{
-		Approvals = new List<Approval>();
 		PriceHistories = new List<PriceHistory>();
 		StartDateChanges = new List<StartDateChange>();
 		FreezeRequests = new List<FreezeRequest>();
-	}
+        Episodes = new List<Episode>();
```

Comment:
Minor comment, but indentation could be tidied up here

Reply:
It's a mixture of tabs and spaces I think which causes it to be funky or non funky depending on which viewer you are using and if it decides a tab is the same width as however many spaces it's used.